frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -188,6 +221,63 @@ class AccountsSummary extends Component<Props, State> {
         );
     }
 
+    private subTotalRow({ coinCode, coinName, coinUnit, balance }: PropsWithChildren<BalanceRowProps>) {
+        const { t } = this.props;
+        const nameCol = (
+            <td data-label={t('accountSummary.total')}>
+                <div className={style.coinName}>
+                    <Logo className={style.coincode} coinCode={coinCode} active={true} alt={coinCode} />
+                    <strong className={style.showOnTableView}>
+                        {t('accountSummary.subtotalWithCoinName', { coinName })}
+                    </strong>
+                    <strong className={style.showInCollapsedView}>
+                        { coinName }
+                    </strong>
+                </div>
+            </td>
+        );
+        if (!balance) {
+            return null;
+        }
+        return (
+            <tr key={`${coinCode}_subtotal`} className={style.subTotal}>
+                { nameCol }
+                <td data-label={t('accountSummary.balance')}>
+                    <span className={style.summaryTableBalance}>
+                        <strong>{balance}</strong>
+                        {' '}
+                        <span className={style.coinUnit}>{coinUnit}</span>
+                    </span>
+                </td>
+                <td data-label={t('accountSummary.fiatBalance')}>
+                    <strong>
+                        <FiatConversion amount={{ amount: balance, unit: coinUnit as accountApi.Coin }} noAction={true} />
+                    </strong>
+                </td>
+            </tr>
+        );
+    }
+
+    private renderAccountSummary() {
+        const { accountsPerCoin } = this;
+        const { totalBalancePerCoin } = this.state;
+        if (!totalBalancePerCoin) {
+            return null;
```

Comment:
this stops rendering until all balances are shown, especially the 'scanned ... addresses' progress indicator is gone. this can simply be removed (i'll do that).

Reply:
done